data/translations/es/state-data.json

```diff
@@ -117,121 +117,122 @@
-    "ip_deadline": "hasta 29 días antes del día de las elecciones",
-    "online_deadline": "hasta 29 días antes del día de las elecciones",
+    "ip_deadline": "martes, 11 de octubre de 2022",
+    "online_deadline": "martes, 11 de octubre de 2022",
     "mail_received_deadline": "",
-    "mail_postmarked_deadline": "debe tener el sello del correo 29 días antes del día de las elecciones",
+    "mail_postmarked_deadline": "debe tener el sello del correo martes, 11 de octubre de 2022",
     "confirm_registration_link": "https://indianavoters.in.gov/"
   },
   "ia": {
     "hp_link": "https://sos.iowa.gov/",
     "registration_link": "https://mymvd.iowadot.gov/Account/Login?ReturnUrl=%2fVoterRegistration",
     "more_info_link": "https://sos.iowa.gov/elections/voterinformation/voterregistration.html",
-    "ip_deadline": "hasta el mismo día de las elecciones",
-    "online_deadline": "hasta 15 días antes del día de las elecciones",
+    "ip_deadline": "martes, 08 de noviembre de 2022",
+    "online_deadline": "lunes, 24 de octubre de 2022",
     "mail_received_deadline": "",
-    "mail_postmarked_deadline": "debe tener el sello del correo 15 días antes del día de las elecciones",
+    "mail_postmarked_deadline": "debe tener el sello del correo lunes, 24 de octubre de 2022",
     "confirm_registration_link": "https://sos.iowa.gov/elections/voterreg/regtovote/search.aspx"
   },
   "ks": {
     "hp_link": "https://sos.ks.gov/elections/elections.html",
     "registration_link": "https://www.kdor.ks.gov/Apps/VoterReg/Default.aspx",
     "more_info_link": "https://sos.ks.gov/elections/voter-information.html",
-    "ip_deadline": "hasta 21 días antes del día de las elecciones",
-    "online_deadline": "hasta 21 días antes del día de las elecciones",
+    "ip_deadline": "martes, 18 de octubre de 2022",
+    "online_deadline": "martes, 18 de octubre de 2022",
     "mail_received_deadline": "",
-    "mail_postmarked_deadline": "debe tener el sello del correo 21 días antes del día de las elecciones",
+    "mail_postmarked_deadline": "debe tener el sello del correo martes, 18 de octubre de 2022",
     "confirm_registration_link": "https://myvoteinfo.voteks.org/voterview/?culture=es-MX",
     "confirm_registration_link_translated": "true"
   },
   "ky": {
     "hp_link": "https://elect.ky.gov/Pages/default.aspx",
     "registration_link": "https://vrsws.sos.ky.gov/ovrweb/",
     "more_info_link": "https://elect.ky.gov/Resources/Pages/Registration.aspx",
-    "ip_deadline": "hasta el cuarto lunes antes del día de las elecciones",
-    "online_deadline": "hasta el cuarto lunes antes del día de las elecciones",
-    "mail_received_deadline": "a más tardar debe ser recibida a más tardar el cuarto lunes antes del día de las elecciones",
-    "mail_postmarked_deadline": "",
+    "ip_deadline": "martes, 11 de octubre de 2022",
+    "online_deadline": "martes, 11 de octubre de 2022",
+    "mail_received_deadline": "",
```

Comment:
Updated from received to postmarked. Please check this. Also remove empty deadline properties.

Reply:
Confirmed, it is postmarked